pkg/proxy/integrations/httpparser/httpparser.go

```diff
@@ -489,11 +490,37 @@ func decodeOutgoingHttp(requestBuffer []byte, clientConn, destConn net.Conn, h *
 		if err != nil {
 			logger.Error("error while matching http mocks", zap.Error(err))
 		}
-
 		if !isMatched {
 			passthroughHost := false
-			for _, host := range models.PassThroughHosts {
-				if req.Host == host {
+			passThrough := models.PassThroughHosts
+			portPassThrough := []models.Filters{}
+			for _, filters := range h.GetProxyHost().Filters {
```

Comment:
Why is the separation being done again it is already done in service layer right..?

Reply:
this is different than the service layer one , as this separates the passthrough on ports for certain url unlike the service layer where we send to ebpf.